loadtest/control/simplecontroller/controller.go

```diff
@@ -178,10 +179,15 @@ func (c *SimpleController) createActions(definitions []actionDefinition) error {
 			return fmt.Errorf("could not find action %q", def.ActionId)
 		}
 
+		if def.RunPeriod < 1 {
```

Comment:
I was actually okay with failing if negative. It's just the zero value that denotes the intention of skipping it. Same result in the end though.

Reply:
Yeah this is actually different. The idea was to allow the user to disable a test. But a value of zero would actually result in a warning. Not sure if that's a good user experience if disabling an action with 0 is something we explicitly support.

Reply:
That's a fair point. The warning idea was to cover the case where the user might have forgotten to add the `RunPeriod` entry in the JSON config, since the default value for a `int` is 0. But I guess that's a pretty rare case which might fall under misusage.

It's probably time to take a decision on this. My suggestion would be:

- Fail the validation if `RunPeriod < 0`.
- Skip the action if `RunPeriod == 0`.
- Add the action otherwise.

Reply:
And also document the fact that if RunPeriod is 0, then the action is skipped.